include/opt-sched/Scheduler/aco.h

```diff
@@ -45,7 +52,9 @@ class ACOScheduler : public ConstrainedScheduler {
   pheromone_t &Pheromone(SchedInstruction *from, SchedInstruction *to);
   pheromone_t &Pheromone(InstCount from, InstCount to);
   double Score(SchedInstruction *from, Choice choice);
-  bool shouldReplaceSchedule(InstSchedule *OldSched, InstSchedule *NewSched);
+  bool shouldReplaceSchedule(InstSchedule *OldSched, InstSchedule *NewSched,
+                             bool IsGlobal);
```

Comment:
`bool` parameters should generally be avoided, as it's hard to tell from the calling code what the bool signifies. I recommend either an `enum class` or using the comment-based named parameters that clang-tidy will check:

```
enum class IS_GLOBAL {
  YES,
  NO,
};

bool shouldReplaceSchedule(..., IS_GLOBAL IsGlobal);
...
shouldReplaceSchedule(Sched, NewSched, IS_GLOBAL::YES);

// Alternatively, in the calling code:
shouldReplaceSchedule(Sched, NewSched, /*IsGlobal=*/true);
// This is checked by clang-tidy, so a typo will be flagged
// (should verify this before relying on it; we might have turned off the check)
```

Reply:
This is a private method, that is only used in two places(and cannot conceivably or meaningfully be used elsewhere).  The purpose of this method was to move complicated logic out of a if statement condition.  I don't want to create a enum for so trivial of a purpose.  I will use the `/*IsGlobal=*/` comment.